apps/E2E/src/common/BasePage.ts

```diff
@@ -72,6 +82,48 @@ export abstract class BasePage {
   /**************** UI Element Interaction Methods ******************/
   /******************************************************************/
 
+  /**
+   * For any given page object, this method automates:
+   *  - Navigating to the page by clicking its component button
+   *  - Waiting for the component page to load
+   *  - Revealing its E2E components IF a boolean flag is passed into the method
```

Comment:
Why do we need a boolean flag to show it's E2E components?

Reply:
Some pages don't have a dedicated section for E2E tests and don't render the switch. If we try and click the unrendered switch, appium won't find the element and throw an error.

Reply:
jw what pages don't have a dedicated E2E section? Do those components have E2E tests running?

Reply:
The pages that only check for baseline E2E crash tests, such as ActivityMonitor, Badge, IconLegacy, don't have dedicated E2E sections. Those tests only check if the page title becomes visible.

Reply:
Let's add that explanation in the comments for the prop description